V2rayNG/app/src/main/kotlin/com/v2ray/ang/service/V2RayVpnService.kt

```diff
@@ -153,8 +151,11 @@ class V2RayVpnService : VpnService(), ServiceControl {
         }
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-            connectivity.requestNetwork(defaultNetworkRequest, defaultNetworkCallback)
-            listeningForDefaultNetwork = true
+            try {
+                connectivity.requestNetwork(defaultNetworkRequest, defaultNetworkCallback)
+            } catch (ignored: Exception) {
+                // ignored
```

Comment:
Let's not ignore this exception by logging it. By the way, if you know the exact exception, we should see if there are other ways to fix the root problem.